packages/context/src/context.ts

```diff
@@ -37,6 +65,13 @@ export class Context {
     this.name = name || uuidv1();
   }
 
+  /**
+   * Get the parent context
+   */
+  get parent() {
```

Comment:
Hmm, is it a good idea to expose the parent context in a public API? Shouldn't users always call Context methods and let them (the Context methods) to handle inheritance traversal?

Reply:
_reposting an older comment: https://github.com/strongloop/loopback-next/pull/2122/files#r245920007_

Hmm, is it a good idea to expose the parent context in a public API? Shouldn't users always call Context methods and let them (the Context methods) to handle inheritance traversal?